### DISCUSSION TITLE
Proper Support for Array Types

### DISCUSSION THREAD

**Discussion by John0x** [directus/directus#14388] [Archive]
Hey, it seems like array types (like text[]) aren't properly supported. Neither on the ui side nor on the api.
Is this intended?

**Comment by azrikahar** [comment]
Technically it is currently intended, and this has been brought up in #12791, but the main "blocker" (not too sure personally as I'm not too involved in this area) would be taking into account for other database vendors as mentioned in this comment: https://github.com/directus/directus/discussions/12791#discussioncomment-2577019
